src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/CronScheduleUtil.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NCrontab;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Services
+{
+    public static class CronScheduleUtil
+    {
+        public static DateTime GetNextScheduledPublishingTime()
+        {
+            var publishReleasesCronSchedule = Environment.GetEnvironmentVariable("PublishReleaseContentCronSchedule");
+            return TryParseCronSchedule(publishReleasesCronSchedule, out var cronSchedule)
+                ? cronSchedule.GetNextOccurrence(DateTime.UtcNow)
+                : DateTime.UtcNow;
+        }
+
+        private static bool TryParseCronSchedule(string cronExpression, out CrontabSchedule cronSchedule)
```

Comment:
'Try' with a boolean returning function is a bit weird. Sounds better as a 'CanParseCronSchedule'

Reply:
This `TryParse` naming is very common and used with the `out` variable.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/exceptions-and-performance#try-parse-pattern

Reply:
Ah I see, fair enough!